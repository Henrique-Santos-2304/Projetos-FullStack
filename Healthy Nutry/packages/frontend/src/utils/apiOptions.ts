export const optionsMealTypes = [
  { value: "", label: "Selecione tipo de refeição" },
  { value: "coffee", label: "Café" },
  { value: "lunch", label: "Almoço" },
  { value: "dinner", label: "Janta" },
];

export const optionsProteins = [
  { value: "", label: "Selecione uma Proteína" },
  { value: "fish", label: "Peixe" },
  { value: "ox", label: "Boi" },
  { value: "pig", label: "Porco" },
  { value: "chicken", label: "Frango" },
  { value: "egg", label: "Ovo" },
];

export const optionsCarbohydrates = [
  { value: "", label: "Selecione um Carboidrato" },
  { value: "rice", label: "Arroz" },
  { value: "potato", label: "Batata" },
  { value: "Yam", label: "Inhame" },
  { value: "noodle", label: "Macarrão" },
  { value: "bread", label: "Pão" },
];

export const optionsVegetables = [
  { value: "", label: "Selecione um Vegetal" },
  { value: "zucchini", label: "Abobrinha" },
  { value: "aubergine", label: "Berinjela" },
  { value: "pumpkin", label: "Abóbora" },
  { value: "chicory", label: "Almeirão" },
  { value: "tomato", label: "Tomate" },
];

export const proteinsChoice = ["Peixe", "Boi", "Porco", "Frango", "Ovo"];
export const carbohydrateChoice = [
  "Arroz",
  "Batata",
  "Inhame",
  "Macarrão",
  "Pão",
];
export const vegetableChoice = [
  "Abobrinha",
  "Berinjela",
  "Abóbora",
  "Almeirão",
  "Tomate",
];
