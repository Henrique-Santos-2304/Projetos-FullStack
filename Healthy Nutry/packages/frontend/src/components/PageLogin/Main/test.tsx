import { renderTheme } from "utils/testRenderTheme";
import { screen } from "@testing-library/react";
import Header from ".";

describe("Header", () => {
  it("should render ", () => {
    const { container } = renderTheme(<Header />);
    expect(container).toMatchSnapshot();
    screen.logTestingPlaygroundURL();
  });
});
