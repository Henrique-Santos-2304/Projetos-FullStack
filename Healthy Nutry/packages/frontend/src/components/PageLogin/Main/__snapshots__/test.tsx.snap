// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render  1`] = `
.c0 {
  width: 100vw;
  height: 100vh;
  text-align: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  width: 28rem;
  height: 28rem;
  position: relative;
}

.c3 {
  width: 6rem;
  height: 6rem;
  position: relative;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c5 {
  font-family: 'Tangerine',cursive;
  color: #b8f706;
  font-size: 9rem;
}

.c6 {
  color: #f8f8f8;
}

<div>
  <main
    class="c0"
  >
    <div
      class="c1"
    >
      <span
        style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
      >
        <img
          class=""
          data-nimg="fill"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
    </div>
    <div
      class="c2"
    >
      <div
        class="c3"
      >
        <span
          style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
        >
          <img
            class=""
            data-nimg="fill"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </div>
      <div
        class="c4"
      >
        <p
          class="c5"
        >
          Web
        </p>
        <p
          class="c5 c6"
        >
          Dev
        </p>
      </div>
    </div>
  </main>
</div>
`;
